### ISSUE TITLE
some suggestions related to R code

### ISSUE THREAD

**Issue by pbiecek** [kadyb/rgugik#12]
Great package, thanks for this work!

Perhaps the following notes will be useful for development and maintenance of this package.

Do not keep URLs inside functions, it will be hard to change them if API changes 
https://github.com/kadyb/rgugik/blob/master/R/pointDTM_get.R#L47
https://github.com/kadyb/rgugik/blob/master/R/pointDTM100_download.R#L47

Nested if/else are hard to read. 
Quite often you can use the return() to exit the function. Smaller number of 'else' statements will make the code more readable.
For example in  
https://github.com/kadyb/rgugik/blob/master/R/geocodePL_get.R

You can use pkgdown and GitHub Actions to automate deployment of pkgdown website (with live examples and vignettes)
This makes it easier to look at individual functions without having to install the package.

**Comment by kadyb** [comment]
@pbiecek 
> Do not keep URLs inside functions, it will be hard to change them if API changes

What do you propose as a solution? For example, a separate file with all service addresses? It seems to me that changing URLs in each case will be troublesome.

> Nested if/else are hard to read.
> Quite often you can use the return() to exit the function. Smaller number of 'else' statements will make the code more readable.

If I understand correctly, this solution will be better?
```r
geocodePL_get = function(address = NULL, road = NULL, ...) {
  if (!is.null(address)) {
    # geocode address
    return(output)
  }
  if (!is.null(road)) {
    # geocode road
    return(output)
  }
  ...
  stop("all inputs are empty")
}
```
CC: @Nowosad

**Comment by pbiecek** [comment]
Ad 1.
Probably the easiest way to do this is to put links as default values for function arguments (like url='http://...'). It is much easier for the user to change argument if needed.

Ad 2.
Yes, Such code is easier to read.
There are fewer indentations, fewer '{' and '}'.
But most importantly, when someone reads a function and see 'return', they know that this is where the code ends and they don't have to trace it further.

**Comment by kadyb** [comment]
Fixed in aef03a3.
